### ISSUE TITLE
[ Xbox360 pad issue ] Controller not Focusing on Game Window in some games

### ISSUE THREAD

**Issue by ekianjo** [ValveSoftware/SteamOS#373]
I have had that issue in games such as Freedom Planet - which is supposed to support controllers. The controller actually is not "working" on the window of the game, only in the SteamOS interface in the background. What is causing this issue ?

**Comment by sharkwouter** [comment]
That's game developers screwing up.

**Comment by ProfessorKaos64** [comment]
Exactly, which is the current issue I have with Dirt: Showdown and creating a profile. Now I am 5 test profiles on the game with no way to delete them. Curses! You'd think you'd be able to delete a profile. We've only had that since. idk..forever?
